ui-v2/app/components/templated-anchor.js

```diff
@@ -1,3 +1,6 @@
+/*eslint ember/require-return-from-computed: "warn"*/
+// TODO: Remove ^
+
```

Comment:
I see that you want to address these later but I think this one would be as simple as an `else { return undefined; }`